subprojects/build-scan-performance/templates.gradle.kts

```diff
@@ -43,20 +43,29 @@ tasks.register<JvmProjectGeneratorTask>("largeJavaProjectWithBuildScanPlugin") {
     linesOfCodePerSourceFile = 150
     rootProjectTemplates = listOf("root")
     subProjectTemplates = listOf("project-with-source")
-    templateArgs = mapOf("fullTestLogging" to true, "failedTests" to true, "projectDependencies" to true, "manyPlugins" to true, "manyScripts" to true)
+    templateArgs = mapOf(
+        "fullTestLogging" to true,
+        "failedTests" to true,
+        "projectDependencies" to true,
+        "manyPlugins" to true,
+        "manyScripts" to true
+    )
 
     doLast {
         // generate script plugins
         val scriptPlugins = 30
         val nesting = 5
-        val groupedScriptIds = ((1..scriptPlugins).groupBy { it % (scriptPlugins/nesting)}.values)
-        val gradleFolder =  File(destDir, "gradle")
+        val groupedScriptIds = ((1..scriptPlugins).groupBy { it % (scriptPlugins / nesting) }.values)
+        val gradleFolder = File(destDir, "gradle")
```

Comment:
:o: could be `destDir.resolve("gradle")`

Same line 67 below

Reply:
👍